### PR TITLE
Removes null crates

### PR DESCRIPTION
**Pull request by Iamgoofball**
They're shit, op, and a crutch. Traitor qms can rain rockets of crates on people. More useful.
:cl: Nanotrasen Customs
del: We've tracked down the source of contraband crates heading to your station including highly illegal and lethal gear, and have dispatched a new wave of customs robots to replace the previous faulty ones.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
Also every time we try to change cargo all I hear is "BUT NULL CRATES" and they're holding us back

**Comment by Togopalis** [comment]
Eat your heart out kevinz000, you have a competition in who can nerf everything the fastest

**Comment by praisenarsie** [comment]
@iamgoofball tbh the only way you're getting them removed is finishing what you started, SHIPPING DATUMS!

**Comment by evsey9** [comment]
yeah i have to agree with  this
null crates are op

**Comment by Buggy123** [comment]
I never roll traitor, but I like the _idea_ of this at least :-1:

**Comment by ExcessiveUseOfCobblestone** [comment]
No, the problem is having so much money you can buy **a bunch** of them hence why it's so important to minimize money production.

If you take nulls then people will just spam guns/swat gear.

**Review by Cyberboss — APPROVED** [review]

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Changelog

**Comment by Slignerd** [comment]
I do agree that non-antags running around in traitor gear, while fun for said non-antags at the time, is a bit odd when you think about it. And by now, it's basically a common occurence.

**Comment by ExcessiveUseOfCobblestone** [comment]
Is it now though after techdisks were removed?

**Comment by Iamgoofball** [comment]
@ExcessiveUseOfCobblestone take your shit elsewhere

I don't care if people spam swat gear or guns, those aren't on the same level as literal uplink items and you fucking know it. Stop arguing in bad faith.

**Comment by PKPenguin321** [comment]
>goof buffs null crates to have 50 TCs of items instead of just being bundles
>wait a few months
>uses the buff to justify a full removal

🤔

**Comment by ExcessiveUseOfCobblestone** [comment]
What exactly is worse than the WT rifle and SWAT armor? A bomb?

Why not just blacklist items instead of removing null crates entirely?

You could also raise the cost and not encourage people to add EZ money machines. Wouldn't that be something?

**Comment by MrDoomBringer** [comment]
literally what is the point of this? you realize currently it takes **40** fucking crates to buy a single NULL, right?

**Comment by praisenarsie** [comment]
@MrDoomBringer have you ever heard of selling minerals or stocks

**Comment by MrDoomBringer** [comment]
@praisenarsie even selling stacks and stacks of plasma (which is a gay use of them, imo) is only gonna net you...3, maybe 4 crates? Ive seen fucktons of nulls ordered and not a single useful item, outside an extra emag or two.

also
>implying cargo's main way of point generation should be funless RNG that takes constant, soul-sucking monitoring and the entire round to get anything substantial

**Comment by Iamgoofball** [comment]
We want to add the PTL to sell excess power for money but people bitch "muh null crate" any time we touch economy so I'm killing null crates @MrDoomBringer

**Comment by Iamgoofball** [comment]
also I'd be killing them for my cargo rework down the line

**Comment by Iamgoofball** [comment]
Raising the cost is a bandaid and telling people to add less features is retarded Cobby

**Comment by MrDoomBringer** [comment]
@Iamgoofball I can respect that, but why not make more OP shit have a smaller chance of coming along, or just making it harder to get?

**Comment by Iamgoofball** [comment]
Adjusting the contents of the null crate is ignoring the problem of letting players order traitor items and doesn't fix the problem of "we have to balance every credit gaining method against null crates"

**Comment by MrDoomBringer** [comment]
Having traitor gear accessible from the crew in /some/ regard other than looting traitors adds a strong amount of gameplay, (ex.  paranoia: how did he get that agent ID? from his uplink or from a nullcrate?) as well as giving players more chances to see how useful some of the less used tator gear can be (promoting less ebow+esword bullshit and more interesting tator builds).

**Comment by MrDoomBringer** [comment]
also @praisenarsie a stack of plasma nets you about 1 nullcrate. have you ever seen a single nullcrate ordered and something good been inside?

**Comment by praisenarsie** [comment]
@MrDoomBringer define 'good'

**Comment by darkpaladin109** [comment]
Oh no, a stack of one of the most common types of ore around gets you "only" 1 crate!

**Comment by arsserpentarium** [comment]
letting players order traitor items isn't a problem.It should be just well gated.
Also if you make more methods of gaining cargo points, there should be some things in cargo, which are OP but expensive.To provide goals for CP gain.
Otherwise, what will you buy,after you set up energy transfer laser and some kind of production line?Infinite pizza crates?

**Comment by MrDoomBringer** [comment]
@darkpaladin109 in my ~1300 connections ive never seen someone sell more than 3 stacks of plasma

**Comment by MrDoomBringer** [comment]
@praisenarsie something that makes buying the nullcrate worth it

**Comment by MrDoomBringer** [comment]
potential solution i mentioned to @Iamgoofball: make NULLS only orderable from an emagged express console. That way the tator QM has to run about the station looking for them/send out their techs to watch out for an asston of nullcrate slanding. 
Secondarily, maybe give the crate a lock (openable by sec/warden) so you're not just dispensing tator gear among the crew, and so that if sec sees it, they have a chance to drag it back to brig for distribution among trusted officers.

**Review by Cyberboss — APPROVED** [review]

**Comment by ma44** [comment]
>This pr again

**Comment by ExcessiveUseOfCobblestone** [comment]
"you're arguing in bad faith!!!"

*PR exists as a strawman to another PR*

Truly amazing the lack of self-awareness.

**Comment by MrDoomBringer** [comment]
@ExcessiveUseOfCobblestone strawman to which pr?
also check your discord jeez

**Comment by ExcessiveUseOfCobblestone** [comment]
@MrDoomBringer at work ^_^

also the laser beam pr #35345

**Comment by MrDoomBringer** [comment]
image
anyways as a fun experiment, i sold a fuckload of plasma (as in, many times more than ive ever seen sold), and saw how many OP and GAMEBREAKING things i could get. anyways, the results were essentially:

- aaa 5 godawful pistols, infinite powerfists and radiojammers, and a **single ebow jesus christ!!!**

**Comment by ExcessiveUseOfCobblestone** [comment]
> doesn't do stocks

> has QM profile pic

:thinking:

**Comment by MrDoomBringer** [comment]
i never said I do stocks, i just believe that they're a pretty damn boring solution for getting money; a problem that has such goddamn potential to have an amazing and fun solution, that hundreds of other games have accomplished.

**Comment by KorPhaeron** [comment]
Having desirable items but limited money seems more fun for players than having infinity money but nothing to buy with it

If it isnt null crates it will be guns or implant spam or something

tl;dr cobby is right

**Comment by Iamgoofball** [comment]
@KorPhaeron Guns and implants are not the same as traitor items my dude

**Comment by Anonymous583** [comment]
Fuck off Goof, go do something good like finishing dwarves

**Comment by Iamgoofball** [comment]
@Anonymous583 itt: angry null crate abuser

**Comment by kevinz000** [comment]
I abuse null crates and I think this is hilarious

**Comment by YakumoChen** [comment]
null crates are a good reason to actually spend TC on the money briefcase
